skimage/segmentation/_expand_labels.py

```diff
@@ -0,0 +1,106 @@
+"""
+expand_labels is derived from code that was
+originally part of CellProfiler, code licensed under BSD license.
+Website: http://www.cellprofiler.org
+
+Copyright (c) 2020 Broad Institute
+All rights reserved.
+
+Original authors: CellProfiler team
+
+"""
+import numpy as np
+from scipy.ndimage import distance_transform_edt
+
+
+def expand_labels(label_image, distance):
```

Comment:
Could we have a default value for `distance` (for example `distance=1`)? This would be more consistent with the rest of our API where most of the time the positional arguments are image arrays and all other arguments are keyword arguments.

Reply:
Thanks for the suggestion, I added a default value of 1 pixel distance.